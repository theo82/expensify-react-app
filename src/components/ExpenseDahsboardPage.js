import React from 'react'
import ExpenseList  from './ExpenseList'
import ExpenseListFilters from './ExpenseListFilters'

export const ExpenseDahsboardPage = () => (
    <div>
       <h1>Header</h1>
       <ExpenseListFilters />
       <ExpenseList />
    </div>
);

export default ExpenseDahsboardPage
