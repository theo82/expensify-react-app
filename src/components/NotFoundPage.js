import React from 'React'
import { Link } from 'react-router-dom'

export const NotFoundPage = () => (
    <div>
        404! - <Link to="/">Home</Link>
    </div>
)

export default NotFoundPage;
