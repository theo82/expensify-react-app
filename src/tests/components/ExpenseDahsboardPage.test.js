import React from 'react';
import { shallow } from 'enzyme';
import { ExpenseDahsboardPage } from '../../components/ExpenseDahsboardPage';

test('should render ExpenseDahsboardPage', () => {
    const wrapper = shallow(<ExpenseDahsboardPage />);
    expect(wrapper).toMatchSnapshot();
});