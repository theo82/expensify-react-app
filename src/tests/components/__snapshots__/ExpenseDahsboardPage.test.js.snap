// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDahsboardPage 1`] = `
<div>
  <h1>
    Header
  </h1>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
